src/public/modules/forms/admin/views/edit-fields.client.modal.html

```diff
@@ -695,13 +695,13 @@ <h2 class="modal-title">
                 <i
                   ng-switch-when="encrypt"
                   class="glyphicon glyphicon-question-sign"
-                  uib-tooltip="Customised email acknowledgement to the respondent."
+                  uib-tooltip="Customize an email acknowledgement to respondents"
                   tooltip-trigger="'click mouseenter'"
                 ></i>
                 <i
                   ng-switch-default
                   class="glyphicon glyphicon-question-sign"
-                  uib-tooltip="Recipients will receive an email confirmation with their submission reference number, and a customised message from you."
+                  uib-tooltip="Customize an email acknowledgement to respondents"
```

Comment:
```suggestion
                  uib-tooltip="Customise an email acknowledgement to respondents"
```
nit: British spelling?